src/sessionOne/BasicTypes.kt

```diff
@@ -1,2 +1,168 @@
 package sessionOne
 
+/**
+ * The main method is the entry point for any application in almost all
+ * programming languages including kotlin, write all your executable code here.
+ * NB : Comments represent non-compilable and non-executable code.
+ */
+fun main() {
+    // Comments represent non-compilable and non-executable code
+    //single line comment
+    /**
+     * Documentation comment
+     */
+
+    /*
+     * Multiline comment
+     * Yes, as you can see, this is not compiled by the kotlin compiler
+     */
+
+    // printing things onto your screen
+    println("Welcome to our lovely course !")
+
+    // basic types
+
+    // an integer is either a +ve or -ve non-fractional 32-bit number
+    val positiveInt = 23222222
+    val integerSize = Int.SIZE_BITS
+    val integerMax: Int = Int.MAX_VALUE
+    println(positiveInt)
+    println(integerSize)
+    println(integerMax)
+
+    // a long is either a +ve or -ve non-fractional 64-bit number
+    val positiveLong = 23222222L
+    val longSize = Long.SIZE_BITS
+    val longMaxValue: Long = Long.MAX_VALUE
+    println(positiveLong)
+    println(longSize)
+    println(longMaxValue)
+
```

Comment:
Fractional numbers are missing, Doubles and Floats.

Reply:
I think we should start first with vars & vals and data types then we teach them comment types. Because at first they don't know what the meaning of compilable and non-compilable is.